circle.yml

```diff
@@ -5,10 +5,6 @@ jobs:
       - image: trzeci/emscripten:sdk-tag-1.37.21-64bit
     steps:
       - checkout
-      - run:
-          name: Init submodules
-          command: |
-            git submodule update --init
```

Comment:
Why were these removed?

Reply:
I think we don't have any submodules anymore, do we? I can remove it from this PR.